install.sh

```diff
@@ -53,13 +87,15 @@ helm upgrade --install --wait --timeout ${HEROKLES_HELM_TIMEOUT:-3m1s} \
   -n ${PROJECT} \
   ${HELM_DEPLOYMENT} \
   ${HELM_DIRECTORY:-herokles/helm} \
+  -f herokles/helm/values-envs.yaml \
```

Comment:
This will not create actual secrets "just" make them available to use in helm chart downstream :+1:

Reply:
Yup, as the good lord intended. Plus the process can write other things into the values-envs, which will not make it into the secrets but will be available for the helm to work with, which is a HUGE benefit compared to previous BS.